.github/workflows/labeler.yml

```
@@ -0,0 +1,11 @@
+name: "PR Labeler"
+on:
+- pull_request_target
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/labeler@main
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

This will remove labels when they are no longer relevant 

```suggestion
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
        sync-labels: true
```